content/en/calendar/_index.md

```diff
@@ -8,14 +8,15 @@ draft: false
 <section class="section">
   <div class="container">
     <div class="row align-items-center">
-        <p>Welcome to the InnerSource Commons Calendar! Our events and working groups are open to everyone because we know that magic happens when people come together to share their experience and knowledge. 
-        <p>A note about this calendar ... If you're using google, our calendar may display in the GMT timezone. To make sure you see events in your own timezone, please open the calendar on an incognito browser window or use a different search engine.
-        <div class="col-md-6 order-1 order-md-2 mb-4 mb-md-0">
-<iframe src="https://calendar.google.com/calendar/embed?src=c_62694f414055ac569e5cb12dafbb0890ca22f3640b177a4b10b53171fbc9bdd4%40group.calendar.google.com" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>        </p>
-      </div>
+        <p>Welcome to the InnerSource Commons Calendar! Our events and working groups are open to everyone because we know that magic happens when people come together to share their experience and knowledge. </p>
+        <div class="col-md-6 order-1 order-md-2 mb-4 mb-md-0" id="calendar-container">
+        </div>
       </div>
     </div>
   </div>
 </section>
 
+<script type="text/javascript">
```

Comment:
Why is this needed?

Reply:
It isn't it was left over from testing. Removed